Content.Client/Ensnaring/Visualizers/EnsnareableVisualizerComponent.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Robust.Shared.Utility;
+
+namespace Content.Client.Ensnaring.Visualizers;
+[RegisterComponent]
+[Access(typeof(EnsnareableVisualizerSystem))]
+public sealed class EnsnareableVisualizerComponent : Component
+{
+    [DataField("sprite")]
+    public string? Sprite;
+
+    [DataField("state")]
+    public string? State;
+}
```

Comment:
AFAIK people told me to do it this way because a visualizer should handle all of this stuff. I think I'd have to add another check to the OnAppearanceChange method to look for the EnsnaringComponent in that case, otherwise I can just access the EnsnareableVisualizerComponent.